ansys/dpf/core/collection.py

```diff
@@ -59,9 +61,43 @@ def __init__(self, dpf_type, collection=None, server: server.DpfServer = None):
         elif hasattr(collection, "_message"):
             self._message = collection._message
             self._collection = collection  # keep the base collection used for copy
+
         else:
             self._message = collection
 
+        if self._type == None:
+            self._type = types(int(self._message.type) + 1)
+
+    @staticmethod
+    def integral_collection(inpt, server: server.DpfServer = None):
+        """Creates a collection of integral type with a list.
+        The collection of integral is the equivalent of an array of
+        data sent server side. It can be used to efficiently stream
+        large data to the server.
+
+        Notes
+        -----
+        Used by default by the ``'Operator'`` and the``'Workflow'`` when a
+        list is connected or returned.
```

Comment:
Notes should be after the Returns section. See https://numpydoc.readthedocs.io/en/latest/format.html